peers/topicpool_test.go

```diff
@@ -328,3 +324,26 @@ func (s *TopicPoolSuite) TestNewTopicPoolInterface() {
 	cacheTP := newCacheOnlyTopicPool(tp, &testTrueVerifier{})
 	s.IsType(&cacheOnlyTopicPool{}, cacheTP)
 }
+
+func (s *TopicPoolSuite) TestIgnoreInboundConnection() {
```

Comment:
What we don't check here is whether an inbound peer is not removed from the server.

Reply:
added such test in a followup commit